### PR TITLE
test: ensure locale files are valid

### PR DESCRIPTION
**Pull request by straker**
Closes: #1660

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [x] Follows the commit message policy, appropriate for next version
- [x] Has documentation updated, a DU ticket, or requires no documentation change
- [x] Includes new tests, or was unnecessary
- [x] Code is reviewed for security by: Stephen

### CODE REVIEW

File: test/test-locales.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+var glob = require('glob');
+var axe = require(path.join(__dirname, '../axe'));
+
+var localeFiles = glob.sync(path.join(__dirname, '../locales/*.json'));
+
+describe('locales', function() {
+	localeFiles.forEach(function(localeFile) {
+		var localeName = path.basename(localeFile);
+		it(localeName + ' should be valid', function(done) {
+			fs.readFile(localeFile, 'utf-8', function(err, localeData) {
```
Comment (stephenmathieson) [inline test/test-locales.js]: Why not use the synchronous version? It'd make this a lot easier to read.

File: test/test-locales.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+				if (err) {
+					done(err);
```
Comment (stephenmathieson) [inline test/test-locales.js]: This will release Zalgo.

Comment (straker) [reply test/test-locales.js]: I'm not sure what you mean

File: test/test-locales.js (hunk 1 above)
Comment (straker) [reply test/test-locales.js]: No particular reason other than I'm use to writing node using async methods

File: test/test-locales.js (hunk 2 above)
Comment (stephenmathieson) [reply test/test-locales.js]: https://blog.izs.me/2013/08/designing-apis-for-asynchrony

TLDR; add a `return` here to avoid calling `done` more than once.

File: test/test-locales.js (hunk 1 above)
Comment (stephenmathieson) [reply test/test-locales.js]: Please refactor this to use synchronous `fs` methods then. It'll hugely help readability.

File: test/test-locales.js (hunk 2 above)
Comment (stephenmathieson) [reply test/test-locales.js]: Better yet, just use `fs.readFileSync` :wink:
